src/compiler/diagnosticMessages.json

```diff
@@ -3519,6 +3519,14 @@
         "category": "Error",
         "code": 7036
     },
+    "Create namespace objects for ECMAScript imports which are neither callable nor constructable, implies `allowSyntheticDefaultImports`.": {

```

Comment:
Create namespace objects when using ECMAScript imports on modules that do not use ECMAScript exports. Implies 'allowSyntheticDefaultImports'.

Reply:
I changed it to `Enables emit interoperability bewteen commonjs and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'.` before reading this comment. 👍 👎 ?